tests/serial/multiple-mappings/MultipleWriteFromMappings.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <solver-interface dimensions="2">
+    <data:scalar name="Pressure" />
+
+    <mesh name="MeshB">
+      <use-data name="Pressure" />
+    </mesh>
+
+    <mesh name="MeshATop">
+      <use-data name="Pressure" />
+    </mesh>
+
+    <mesh name="MeshABottom">
+      <use-data name="Pressure" />
+    </mesh>
+
+    <participant name="A">
+      <use-mesh name="MeshATop" provide="yes" />
+      <use-mesh name="MeshABottom" provide="yes" />
+      <read-data name="Pressure" mesh="MeshATop" />
+      <read-data name="Pressure" mesh="MeshABottom" />
+    </participant>
+
+    <participant name="B">
+      <use-mesh name="MeshB" provide="yes" />
+      <use-mesh name="MeshATop" from="A" />
+      <use-mesh name="MeshABottom" from="A" />
+      <write-data name="Pressure" mesh="MeshB" />
+      <mapping:nearest-neighbor
+        direction="write"
+        from="MeshB"
+        to="MeshATop"
+        constraint="consistent" />
+      <mapping:nearest-neighbor
+        direction="write"
+        from="MeshB"
+        to="MeshABottom"
+        constraint="consistent" />
+    </participant>
```

Comment:
@uekerman I'm not sure if this test is supposed to work as well. Currently (as before) it fails. I guess the main reason is that we call 
https://github.com/precice/precice/blob/7618982a33a340a151c6964430799048314d3bdd/src/precice/config/ParticipantConfiguration.cpp#L535
twice using the same data context (although we do the same for the read case, not completely sure about this).

Reply:
> I guess the main reason is that we call

Yes, that's definitely the case.

> although we do the same for the read case where is works, not completely sure about this

No, in this case we configure another data context (namely the read data context, which is different for each mesh and data). The config here simply doesn't work with the `dataContext`

Reply:
Such a case should work, yes. Good example. We need to extend the implementation here.